client/src/test/scala/drt/client/components/ApiFeedStatusTest.scala

```diff
@@ -16,21 +17,17 @@ class ApiFeedStatusTest extends AnyWordSpec with Matchers {
     val beforeNow = "2022-05-31T11:30"
     val afterNow = "2022-05-31T12:30"
 
-    val landedWithNoSources = ArrivalGenerator.apiFlight(schDt = beforeNow,
-      passengerSources = Map(ApiFeedSource -> Passengers(actual = Option(100), transit = None)))
-    val landedWithNoPax = ArrivalGenerator.apiFlight(schDt = beforeNow,
-      passengerSources = Map(ApiFeedSource -> Passengers(actual = None, transit = None)))
-    val landedWithLiveSource = ArrivalGenerator.apiFlight(schDt = beforeNow,
-      passengerSources = Map(LiveFeedSource -> Passengers(actual = Option(100), transit = None)), feedSources = Set(LiveFeedSource))
-    val notLanded = ApiFlightWithSplits(ArrivalGenerator.apiFlight(schDt = afterNow,
-      passengerSources = Map(ApiFeedSource -> Passengers(actual = Option(100), transit = None))), Set())
+    val landedWithNoSources = ArrivalGenerator.arrival(schDt = beforeNow, totalPax= Option(100), transPax = None, feedSource = LiveFeedSource)
```

Comment:
should this be ForecastFeedSource?

Reply:
The feed source doesn't make any difference to this test